wagtail/wagtailadmin/utils.py

```diff
@@ -175,6 +177,13 @@ def test(user):
     return user_passes_test(test)
 
 
+def get_page_models():
+    """List all models subclassing Wagtail Page."""
+    for model in django.apps.apps.get_models():
+        if Page in inspect.getmro(Page):
```

Comment:
It could even be better by using the Django API:
```python
if Page in model._meta.parents:
```
or using the simplest way to do it:
```python
if issubclass(model, Page):
```
I prefer the latter, as it looks clearer.